src/app.ts

```diff
@@ -1,9 +1,12 @@
 import express from 'express';
-import PinoHttp from 'pino-http';
+import pinoHttp from 'pino-http';
```

Comment:
Thanks for correcting this nuance. I'm not sure which example I followed or if it was a guess.

I see this in the source at https://github.com/pinojs/pino-http/blob/c9ce2ac15cb1ad3bb72598c4034b46cce5333ea2/index.d.ts#L66
```
export default PinoHttp;
export { PinoHttp as pinoHttp };
```
But then above at https://github.com/pinojs/pino-http/blob/c9ce2ac15cb1ad3bb72598c4034b46cce5333ea2/index.d.ts#L15, PinoHttp is a function, not a type nor interface nor class:
```
declare function PinoHttp(opts?: Options, stream?: pino.DestinationStream): HttpLogger;

declare function PinoHttp(stream?: pino.DestinationStream): HttpLogger;
```
But wait, it's a function of type `HttpLogger` which is an `interface`:
```
export interface HttpLogger {
...
```
I'll assume this makes sense to you and accept the change in capitalization in good faith.